tests/test_scrapy_settings.py

```diff
@@ -0,0 +1,121 @@
+import attr
+import pytest
+from pytest_twisted import inlineCallbacks
+from scrapy import Request, Spider
+from scrapy.settings import Settings
+from web_poet.pages import Injectable, ItemWebPage
+
+from scrapy_poet.page_input_providers import (
+    PageObjectInputProvider,
+    provides,
+)
+
+from tests.utils import crawl_single_item, HtmlResource
+
+
+class ProductHtml(HtmlResource):
+
+    html = """
+    <html>
+        <div class="breadcrumbs">
+            <a href="/food">Food</a> / 
+            <a href="/food/sweets">Sweets</a>
+        </div>
+        <h1 class="name">Chocolate</h1>
+        <p>Price: <span class="price">22€</span></p>
+        <p class="description">The best chocolate ever</p>
+    </html>
+    """
+
+
+@provides(Settings)
```

Comment:
Is this one required?
Shouldn't it be by default now?

Reply:
I think @ivanprado has proposed the same thing: create some providers for the types we're injecting... Let me see how it looks like.

Reply:
I've updated the pull request including some code to reduce boilerplate required by those Scrapy-provided dependencies.